packages/notification-services-controller/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.0.1]
+
 ### Changed
 
 - Bump `@metamask/base-controller` from ^8.0.0 to ^8.0.1 ([#5722](https://github.com/MetaMask/core/pull/5722))
```

Comment:
Checking if this line is supposed to be here? Seems to be a release PR?

Reply:
Yes it should

Reply:
This was appended from a different PR release:
https://github.com/MetaMask/core/pull/5722/files#diff-de1109262ba78bf35e53809e03a72f22c3e0d5aef47ab6cf9fdb02a5d8560a42

Since this is also a patch update, I don't think this should impact this PR as a patch release.